calculadora_do_cidadao/fields.py

```diff
@@ -13,7 +13,23 @@ def deserialize(cls, value: str) -> Decimal:
         return super().deserialize(value.replace(",", "."))
 
 
+class UsDecimalField(fields.DecimalField):
+    """ Field for reading decimal fields in American format. """
+
+    @classmethod
+    def deserialize(cls, value: str) -> Decimal:
+        """Deserialize decimals using a comma as a decimal separator."""
+        value = value or ""
+        return super().deserialize(value)
+
+
 class DateField(fields.DateField):
     """Field for mmm/yyyy dates."""
 
     INPUT_FORMAT = "%b/%Y"
+
+
+class IsoDateField(fields.DateField):
+    """Field for mmm/yyyy dates."""
```

Comment:
This docstring doesn't match the field definition.

Reply:
Forgot to change it! LOL